tests/integration/test_task_events_mgr.py

```diff
@@ -42,3 +43,33 @@ async def test_process_job_logs_retrieval_warns_no_platform(
         warning = caplog.records[-1]
         assert warning.levelname == 'WARNING'
         assert 'Unable to retrieve' in warning.msg
+
+
+async def test_process_message_no_repeat(
+    one_conf: Fixture, flow: Fixture, scheduler: Fixture, run: Fixture
+):
+    """Don't log received messages if they are found again when."""
+    reg: str = flow(one_conf)
+    schd: 'Scheduler' = scheduler(reg, paused_start=True)
+    message: str = 'The dead swans lay in the stagnant pool'
```

Comment:
Charming 😆

Reply:
(Hitchikers Reference - The worst poetry in the Universe)